#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_sysctl_8.dpatch by  <csmall@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: -p flag has OPTIONAL filename (#297144)
## DP: cleanup ssysctl man page #282168

@DPATCH@
diff -urNad procps-3.2.7~/sysctl.8 procps-3.2.7/sysctl.8
--- procps-3.2.7~/sysctl.8	2006-06-26 09:44:20.000000000 +1000
+++ procps-3.2.7/sysctl.8	2006-06-26 12:42:08.000000000 +1000
@@ -10,76 +10,108 @@
 .SH NAME
 sysctl \- configure kernel parameters at runtime
 .SH SYNOPSIS
-.B "sysctl [-n] [-e] variable ..."
+.B sysctl
+.RB [ \-n ]
+.RB [ \-e ]
+.I variable
+\&...
 .br
-.B "sysctl [-n] [-e] [-q] -w variable=value ..."
+.B sysctl
+.RB [ \-n ]
+.RB [ \-e ]
+.RB [ \-q ]
+.B \-w
+.IR variable = value
+\&...
 .br
-.B "sysctl [-n] [-e] [-q] -p <filename>"
+.B sysctl
+.RB [ \-n ]
+.RB [ \-e ]
+.RB [ \-q ]
+.B \-p
+.RI [ filename ]
 .br
-.B "sysctl [-n] [-e] -a"
+.B sysctl
+.RB [ \-n ]
+.RB [ \-e ]
+.B \-a
 .br
-.B "sysctl [-n] [-e] -A"
+.B sysctl
+.RB [ \-n ]
+.RB [ \-e ]
+.B \-A
 .SH DESCRIPTION
 .B sysctl
 is used to modify kernel parameters at runtime.  The parameters available
 are those listed under /proc/sys/.  Procfs is required for 
-.B sysctl(8)
+.B sysctl
 support in Linux.  You can use
-.B sysctl(8)
+.B sysctl
 to both read and write sysctl data.
 .SH PARAMETERS
 .TP
-.B "variable"
+.I variable
 The name of a key to read from.  An example is kernel.ostype.  The '/'
 separator is also accepted in place of a '.'.
 .TP
-.B "variable=value"
-To set a key, use the form variable=value, where variable is the key and
-value is the value to set it to.  If the value contains quotes or characters
+.IR  variable = value
+To set a key, use the form
+.IR  variable = value
+where
+.I variable
+is the key and
+.I value
+is the value to set it to.  If the value contains quotes or characters
 which are parsed by the shell, you may need to enclose the value in double
-quotes.  This requires the -w parameter to use.
+quotes.  This requires the
+.B \-w
+parameter to use.
 .TP
-.B "-n"
+.B \-n
 Use this option to disable printing of the key name when printing values.
 .TP
-.B "-e"
+.B \-e
 Use this option to ignore errors about unknown keys.
 .TP
-.B "-N"
+.B \-N
 Use this option to only print the names. It may be useful with shells that
 have programmable completion.
 .TP
-.B "-q"
+.B \-q
 Use this option to not display the values set to stdout.
 .TP
-.B "-w"
+.B \-w
 Use this option when you want to change a sysctl setting.
 .TP
-.B "-p"
+.B \-p
 Load in sysctl settings from the file specified or /etc/sysctl.conf if none given.
 Specifying \- as filename means reading data from standard input.
 .TP
-.B "-a"
+.B \-a
 Display all values currently available.
 .TP
-.B "-A"
+.B \-A
 Display all values currently available in table form.
 .SH EXAMPLES
-.TP
-/sbin/sysctl -a
-.TP
-/sbin/sysctl -n kernel.hostname
-.TP
-/sbin/sysctl -w kernel.domainname="example.com"
-.TP
-/sbin/sysctl -p /etc/sysctl.conf 
+/sbin/sysctl \-a
+.br
+/sbin/sysctl \-n kernel.hostname
+.br
+/sbin/sysctl \-w kernel.domainname="example.com"
+.br
+/sbin/sysctl \-p /etc/sysctl.conf 
 .SH FILES
 .I /proc/sys
+.br
 .I /etc/sysctl.conf
 .SH SEE ALSO
 .BR sysctl.conf (5)
 .SH BUGS
-The -A parameter behaves just as -a does.
+The 
+.B \-A
+parameter behaves just as 
+.B \-a
+does.
 .SH AUTHOR
 George Staikos, <staikos@0wned.org>
 
